src/node.rs

```diff
@@ -1316,7 +1445,7 @@ mod tests {
             node.msg_log.add_message(message);
         }
 
-        let seal = node.build_seal(&mut state, vec![1, 2, 3], head).unwrap();
+        let seal = node.build_seal(&state, vec![1, 2, 3], head).unwrap();
```

Comment:
This change should be in the commit where you change the signature for this method